### ISSUE TITLE
false positive on _asdict() member of collections.namedtuple instances

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#141]
Originally reported by: **Anonymous**

---

4th line in code below creates false positive

```
#!python

import collections
Point = collections.namedtuple('Point', ['x', 'y'])
pp = Point(2, 3)
pp_dict_0 = pp._asdict() # pylint: Instance of 'Point' has no '_asdict' member
print(pp_dict_0)
pp_dict_1 = pp.__dict__ # no complaints by pylint here
print(pp_dict_1)

```

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/141
